client/client.go

```diff
@@ -141,6 +96,7 @@ type RPCClient interface {
 	//     on your needs.
 	WithCallerComponent(callerComponent caller.Component) RPCClient
 
+	region.Client
```

Comment:
How about creating a client that contains the interfaces related to router functionality?

Reply:
You mean using the term "router" rather than "region"?